packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -27,9 +27,42 @@ ${chalk.dim('Options:')}
       --orderBy id|deploymentID|amount|priority|...|updatedAt       Order actions by a specific field (default: id)
       --orderDirection asc|desc                                     Order direction (default: desc)
       --first [N]                                                   Fetch only the N first records (default: all records)
+      --fields [field1,field2]                                      Comma-separated names of the fields to display
```

Comment:
I'd add a `,...` in the array to indicate it can be an n length array. Also, since I believe it is required that there are no spaces it would be useful to indicate that as well. 

So it would then be something like: 
```--fields [field1,field2,...] Comma-separated names of the fields to display (no spaces allowed between fields)```

Reply:
Fixed in 9ea1832.